content/user/ui/report/Org-stats.md

```diff
@@ -0,0 +1,51 @@
+<!--
+title: "Organization Statistics"
+description: "Overview of organization statistics charts"
+tags: "TeamServer licenses applications servers organizations report"
+-->
+
+For a summary of administrative statistics on your Contrast installation, look no further than your Organization Statistics. 
+
+Go go to the Reports page via the User Menu to find information about licensing and breakdowns across various data points for onboarded applications; deployed servers and how they’re being utilized; and licensing to understand utilization and expirations. A collection of filters and charts let you decide which data you want to see and how you want to see it. 
```

Comment:
The charts are listed out of order and there are references to pieces that you don't see here until you delve into the Org Stats page . "Go to the main Reports page via the User menu to find informational widgets on licensing utilization, breakdowns across various data points for onboarded applications, as well as deployed servers and how they're being utilized. Filters allow you to determine which data you'd like to view at a glance."